README.md

```diff
@@ -1 +1,135 @@
 # crypto
+
+## First Time Setup
+
+1. Install PyEnv to manage python installations. `brew install pyenv`
```

Comment:
Maybe also mention `curl https://pyenv.run | bash` for non-Mac users or point to https://github.com/pyenv/pyenv-installer

Reply:
Fixed! Thanks!